ci-operator/config/stackrox/stackrox/stackrox-stackrox-ci-perf-hacks.yaml

```diff
@@ -0,0 +1,166 @@
+binary_build_commands: .openshift-ci/dispatch.sh binary-build-commands
+build_root:
```

Comment:
```suggestion
binary_build_commands: .openshift-ci/dispatch.sh binary-build-commands
base_images:
  golang-1.17.2:
    name: golang
    namespace: stackrox
    tag: 1.17.2
build_root:
```

Reply:
After this we should point in our golang version check script that this needs to be changed too https://github.com/stackrox/stackrox/blob/master/EXPECTED_GO_VERSION#L1